odoo/tools/translate.py

```diff
@@ -553,10 +553,20 @@ def __init__(self, source):
         self.source = pycompat.csv_reader(source, quotechar='"', delimiter=',')
         # read the first line of the file (it contains columns titles)
         self.fields = next(self.source)
+        self.fields += list(set(["imd_name", "imd_model"]) - set(self.fields))
```

Comment:
Why the concatenation? Why not just
```suggestion
        self.fields = list(set(next(self.source)) | {"imd_name", "imd_model"})
```

Reply:
Because order is important. If I convert `next(self.source)`, I potentially lose the order for the `zip` following

Reply:
Ah yes, good point.